arcade/gui/constructs.py

```diff
@@ -25,50 +31,66 @@ def __init__(
         width: float,
         height: float,
         message_text: str,
-        buttons=("Ok",),
-        callback=None
-    ):
+        background: arcade.Texture = arcade.load_texture(
+            ":resources:gui_basic_assets/window/grey_panel.png"
+        ),
+        buttons: Tuple[str, ...] = ("Ok",),
+        disappear: bool = False,
+        disappear_time: float = 3,
+        callback: Callable = None,
+    ) -> None:
         super().__init__(size_hint=(1, 1))
-        self._callback = callback  # type: ignore
-
-        space = 10
 
-        # setup frame which will act like the window
-        frame = self.add(UIAnchorLayout(width=width, height=height))
-        frame.with_padding(all=space)
-
-        self._bg_tex = arcade.load_texture(
-            ":resources:gui_basic_assets/window/grey_panel.png"
-        )
-        frame.with_background(texture=self._bg_tex)
+        # Store a few variables needed for this class
+        self._callback: Callable = callback  # type: ignore
+        self._disappear: bool = disappear
+        self._disappear_time_counter: float = disappear_time
+        padding_size = 10
```

Comment:
This should probably be turned into another constructor kwarg with a default value.